docs/browser-usage.md

```diff
@@ -22,17 +24,38 @@ module.exports = {
 
 In case of cypress, you can adapt the same webpack config trick just by applying the [cypress-webpack-preprocessor](https://github.com/cypress-io/cypress-webpack-preprocessor) plugin.
 
+## Angular and similar
+
 However, in some TypeScript projects like the ones using Angular, which AoT compiler requires that a full `*.ts` file is provided instead of just a `*.js` and `*.d.ts` files, to use this shim we have to simply set up our TypeScript configuration in `tsconfig.json` to use this file instead of a normal TypeGraphQL module:
 
 ```json
 {
   "compilerOptions": {
     "baseUrl": ".",
     "paths": {
-      "type-graphql": ["./node_modules/type-graphql/dist/browser-shim.ts"]
+      "type-graphql": ["node_modules/type-graphql/dist/browser-shim.ts"]
+    }
+  }
+}
+```
+
+## Next.js and similar
+
+When using the shim with Next.js as a dedicated frontend server be aware that Next has pre-renders on the server. This means that in development mode the `webpack: {}` config in `next.config.js` is skipped and full `type-graphql` is bundled. But you still need to handle some webpack rewiring for the client bundling which still happens with webpack both in development and in production mode.
+
+The easiest way is to accomplish this is also done in  `tsoconfig.json`—add the same keys like in the example before to `compilerOptions`:
+
+```json
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "type-graphql": ["node_modules/type-graphql/dist/browser-shim.ts"]
     }
   }
 }
```

Comment:
I think it's enough to reference the `tsconfig.json` from above as its 1:1 the same

Reply:
Yes if we remove the headings again one `paths` config code block should be enough. It did it 2x because some people (like me) just ignore anything written about 'Angular`, so I totally missed that piece of info for quite some time.